### DISCUSSION TITLE
zh/guide/advanced/rapid_upload

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#284] [Announcements]
# zh/guide/advanced/rapid_upload

介绍 秒传 就是两个云盘之间的文件可以快速相互上传，不用等待，下面介绍一下都有哪些可以相互秒传 ^v3.29.0新增^ ==有个前提 就是A网盘需要秒传的文件，在B网盘已经有了，才可以进行秒传否则就是正常上传== 阿里云盘 115 百度云盘 天翼云盘 迅雷云盘 PikPak 谷歌云盘 OndDrive 本地存储 :------: :----------...

https://alist.nn.ci/zh/guide/advanced/rapid_upload.html

**Comment by lianshuai0403** [comment]
用什么工具挂载才能秒传？alist桌面版吗

**Comment by magiccode1412** [comment]
测试过本地到OneDrive，不能秒传，我从OneDrive复制文件到服务器，再把文件从服务器复制到OneDrive另一个目录，是正常上传的

**Reply by yyyytawa** [reply]
PS: OneDrive不支持秒传(至少官方网页端没有)

**Comment by GSMAQ** [comment]
求教，
注(3)：阿里云盘Open 和 天翼云盘 ：都需要在存储配置中将配置中的秒传选项打开，否则无法秒传
这个秒传选项，我怎么找不到在哪里打开

**Reply by magiccode1412** [reply]
添加存储页面的最下面

**Reply by GSMAQ** [reply]
好的，谢谢，我已经找到了，需要在添加存储里面选择天翼云盘客户端才会有秒传选项，选择天翼云盘的话是没有秒传开关的。

**Comment by Mosney** [comment]
没太看懂这表格是竖列表示来源网盘、横行表示目标网盘，还是相反

**Reply by anwen-anyi** [reply]
左侧是来源网盘，最上面那一排是目标网盘。

**Comment by daiyang94815** [comment]
可以让夸克秒传至阿里云盘吗？

**Reply by anwen-anyi** [reply]
> 可以让夸克秒传至阿里云盘吗？

不能

**Reply by heli9913** [reply]
onedrive可以秒传到115，但是好像用的本地带宽？

**Comment by heli9913** [comment]
测试可用onedrive可以秒传到115，但是好像用的本地带宽？

**Comment by Choufish** [comment]
请问收否能够对复制任务并行处理的数量进行限制啊，全部开始给路由器干崩溃了。

**Reply by anwen-anyi** [reply]
可能你需要改这俩，具体的琢磨一下~ 

https://alist.nn.ci/zh/config/configuration.html#max-connections
```json
  "max_connections": 0,
```


https://alist.nn.ci/zh/config/configuration.html#tasks
```json
  "tasks": {
    "download": {
      "workers": 5,
      "max_retry": 1
    },
    "transfer": {
      "workers": 5,
      "max_retry": 2
    },
    "upload": {
      "workers": 5,
      "max_retry": 0
    },
    "copy": {
      "workers": 5,
      "max_retry": 2
    }
  }

```

**Comment by zyfnbsl** [comment]
支持夸克就更好了

**Comment by zyfnbsl** [comment]
使用本地存储秒传到百度网盘没效果啊

**Comment by zyfnbsl** [comment]
求助，我从用软件给视频添加外挂字幕后，从本地存储复制到阿里云盘，没有实现秒传，特别慢，没添加外挂字幕前又能秒传，不知道咋回事

**Reply by zyfnbsl** [reply]
或者对视频有所改动就不能秒传

**Reply by anwen-anyi** [reply]
> 或者对视频有所改动就不能秒传

肯定，云盘都没这个文件怎么秒传

**Reply by zyfnbsl** [reply]
我是从本地存储往阿里云盘传的

**Reply by zyfnbsl** [reply]
你的意思是目标盘必须有这个文件吗

**Reply by zyfnbsl** [reply]
目标盘如果有这个文件，那秒传有什么意义

**Reply by anwen-anyi** [reply]
> 目标盘如果有这个文件，那秒传有什么意义

不是你的盘里没有这个文件 是网盘系统里面没有
而且网盘里面没这个文件凭空给你造一个吗…

**Reply by zyfnbsl** [reply]
那我误解这个功能了，还以为能从本地存储秒传到阿里云盘

**Reply by anwen-anyi** [reply]
> 那我误解这个功能了，还以为能从本地存储秒传到阿里云盘

是能秒传 是有条件的

**Reply by zyfnbsl** [reply]
好的，谢谢你的回复！

**Reply by zyfnbsl** [reply]
这样子果然可以，先用上传功能，上传到阿里云盘，然后再从本地存储秒传到阿里云盘，🤭👏👏👏👏👏👏

**Comment by lmx4727** [comment]
请问123和115可不可以秒传

**Reply by anwen-anyi** [reply]
目前不行

**Reply by lmx4727** [reply]
好的，谢谢

**Comment by VicissitudeYW** [comment]
请问一下，复制文件时出现这个报错，是服务器限制还是网盘端（阿里云盘Open → 115）的问题？
（之前用秒传复制了很多大文件备份到另一个网盘）
```
失败
getting src object
upload reach the limit
```

**Comment by liyev** [comment]
两个不同的115账号之间似乎完全传不了，复制之后显示成功，但是文件并没有过去

**Comment by mspmjl** [comment]
getting src object
http request failure, err:http request [https://cn-beijing-data.aliyundrive.net/DcAcDnrS%2F2275279%2F636cfa9d16c676c4a88149ec8

**Comment by mspmjl** [comment]
getting src object
http request failure, err:http request [https://cn-beijing-data.aliyundrive.net/DcAcDnrS%2F2275279%2F636cfa9d16c676c4a88149ec8
阿里云到115 报这个错是为啥

**Comment by Krystal0908** [comment]
百度网盘的资料可以秒传到阿里云盘吗？不行的话可以秒传到哪些网盘？

**Comment by ody778** [comment]
大于 5G 的文件都没用

**Comment by Neon1998** [comment]
秒传失败报错
getting src object

userid/filesize/target/pickcode invalid: userid/filesize/target/pickcode/ invalid

**Comment by xiaoyi0805** [comment]
天翼能秒传到123云盘吗

**Comment by cuibo365** [comment]
阿里云秒传到115需要阿里云的vip么，我怎么是先下载到本地才能秒传到115
